### ISSUE TITLE
Inconsistency in tags/labels between Influxdb(network plugin) and Prometheus endpoints (host vs instance)

### ISSUE THREAD

**Issue by akilan27** [collectd/collectd#2932]
*   Version of collectd: 5.8.0-4
*   Operating system / distribution: Centos

## Expected behavior

That output from network plugin and write_prometheus plugin will be consistent as much as possible.

## Actual behavior

The tags/labelling are different. In Influxdb the data is stored as:
`cpu_value,host=dev1.local,instance=11,type=percent,type_instance=interrupt value=96.1660218079493 `

whereas in Prometheus the same line is:
`collectd_cpu_percent{cpu="11",type="interrupt",instance="dev1.local"} 96.1660218079493 1538151439962`

Basically hostname is stored under different names, and instance means different things. Is there a reason for this? This makes it difficult to adapt Grafana dashboards for use with Influx and Prometheus.

Thanks for pointers.

**Comment by rpv-tomsk** [comment]
>The tags/labelling are different. In Influxdb the data is stored as:
>cpu_value,host=dev1.local,instance=11,type=percent,type_instance=interrupt 
>value=96.1660218079493

>That output from network plugin and write_prometheus plugin will be consistent as much as possible.

Just to clarify things:
The `network` plugin send data in binary form. Provided representation is built by Influxdb implementation.

If you want to find explanation why `write_prometheus` provide data in its current form, you can ask / wait for answer from @octo.
